src/percy-agent-client/client-options.ts

```diff
@@ -1,6 +1,7 @@
 export interface ClientOptions {
   clientInfo?: string,
   environmentInfo?: string,
+  postSnapshotDirectly?: boolean,
```

Comment:
This is not the best name, in terms of clarity, but I haven't come up with anything better yet.

Reply:
* `handleSnapshotPosting`
* `handleAgentCommunication`

Reply:
Since we're going to be doing the healthchecking ourselves too, we can go with `handleAgentCommunication`, I guess.